pkg/runtime/wfengine/backend.go

```diff
@@ -266,7 +321,7 @@ func (*actorBackend) Stop(context.Context) error {
 
 // String displays the type information
 func (be *actorBackend) String() string {
-	return fmt.Sprintf("dapr.actors/v1-alpha")
+	return "dapr.actors/v1-beta"
```

Comment:
Should this be backported into 1.12?

Reply:
Ideally yes but I don’t think it’s important.